mobile/src/lib/cozy-helper.js

```diff
@@ -83,10 +83,10 @@ export function refreshFolder (dispatch, getState) {
 }
 
 export const onError = (dispatch, getState) => (err) => {
-  if (err.message === clientRevokedMsg) {
+  if (err.message === clientRevokedMsg || err.error === 'code=400, message=Invalid JWT token') {
```

Comment:
PouchDB encapsulates network error message in another error with a different generic message. The real error is available in `err.error`.